packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -979,8 +979,7 @@ describe('Collection import', function () {
       );
       const documentsText = await messageElement.getText();
       expect(documentsText).to.not.equal('1 – 20 of 16116');
-      const result = await getFirstListDocument(browser);
-      expect(result._id).to.exist;
```

Comment:
This wasn't what was flaking, but it could if the test aborts instantly, so we should remove it.